README.md

```diff
@@ -2,23 +2,89 @@
-We are currently working toward an alpha version to be used in conjunction with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones) for information about current and future milestones.
+`cri-containerd` is in alpha. This release is for use with Kubernetes 1.7. See
+the [roadmap](./docs/proposal.md#roadmap-and-milestones)
+for information about current and future milestones.
+## Getting started
+### Binary Dependencies and Specifications
+The current release of cri-containerd has been tested with:
+* [containerd](https://github.com/containerd/containerd) v0.2.3-1098-g8ed1e24
+* [runc](https://github.com/opencontainers/runc) v1.0.0-rc3
+* [CNI](https://github.com/containernetworking/cni) v0.4.0
+
+As containerd and runc move to their respective general availability releases,
+we will do our best to rebase/retest `cri-containerd` with these releases on a
+weekly/monthly basis. Similarly, given that `cri-containerd` uses the Open
+Container Initiative (OCI) [image](https://github.com/opencontainers/image-spec)
+and [runtime](https://github.com/opencontainers/runtime-spec) specifications, we
+will also do our best to update `cri-containerd` to the latest releases of these
+specifications as appropriate.
+### Install Dependencies
+1. Install runc dependencies.
+* runc requires installation of the libsecomp development library appropriate for your distribution. `libseccomp-dev` (Ubuntu, Debian) / `libseccomp-devel` (Fedora, CentOS, RHEL). On releases of Ubuntu <=Trusty and Debian <=jessie a backport version of
+`libsecomp-dev` is required. See [travis.yml](XXXX) for an example on
+trusty.
+2. Install containerd dependencies.
+* containerd requires installation of `btrfs-tools`. `btrfs-tools`(Ubuntu, Debian) / `btrfs-progs-devel`(Fedora, CentOS, RHEL)
+3. Install and setup a go1.8.x development environment.
+4. Make a local clone of this repository.
+5. Install binary dependencies by running the following command from your cloned `cri-containerd/` project directory:
+```shell
+# Note: install.deps installs the above mentioned runc, containerd, and CNI
+# binary dependencies. install.deps is only provided for general use and ease of
+# testing. To customize `runc` and `containerd` build tags and/or to configure
+# `cni`, please follow instructions in their documents.
+make install.deps
+```
+### Build and Install cri-containerd
+To build and install `cri-containerd` enter the following commands from your `cri-containerd` project directory:
+```shell
+make
+sudo make install
+```
+### Validate Your cri-containerd Setup
+Another Kubernetes incubator project called [cri-tools](https://github.com/kubernetes-incubator/cri-tools)
+includes programs for exercising CRI implementations such as `cri-containerd`.
+More importantly, cri-tools includes the program `critest` which is used for running
+[CRI Validation](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-validation.md) and benchmark tests.
```

Comment:
running [CRI Validation Test](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-validation.md).

Get rid of "benchmark tests". It's part of "CRI Validation Tests". We may also use `make test-cri` with appropriate arguments to run benchmark in the future.